packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/index.md

```diff
@@ -33,7 +32,7 @@ Set up your [Okta org](/docs/concepts/okta-organizations/). The Okta command-lin
 1. If you don't already have a free Okta developer account:
    1. Open your terminal.
    [[style="list-style-type:lower-alpha"]]
-   1. Run `okta register`, and enter your first name, last name, email address, and country.
+   1. Run `okta register`, and enter your first name, family name, email address, and country.
```

Comment:
```suggestion
   1. Run `okta register`, and enter your first name, last name, email address, and country.
```

Reply:
Is Acrolinx penalizing you for using "last name"? I thought we fixed that, but maybe only on the product side?